lib/bugsnag/utility/circular_buffer.rb

```diff
@@ -2,30 +2,57 @@ module Bugsnag::Utility
   class CircularBuffer
     include Enumerable
 
+    # @return [Numeric] the current maximum allowable number of items
     attr_reader :max_items
 
+    ##
+    # Creates a circular buffer
+    #
+    # @api private
```

Comment:
If users aren't exposed to the `CircularBuffer` in any direct way, move this `@api private` comment to the class (right before line 2). Also related, add a brief comment comment for the `CircularBuffer` class.